src/generator/templates/sample.njk

```diff
@@ -0,0 +1,77 @@
+{% if standalone %}
+{% include "license.njk" %}
+
+// THIS FILE IS AUTO-GENERATED
```

Comment:
what generates this file?

Reply:
Hah, this is hand written - I want the files *it generates* to have that comment.